### PR TITLE
Cleanup failing tests

### PR DESCRIPTION
**Pull request by NJtwentyone**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -189,12 +189,7 @@
       <version>${javax.servlet-api.version}</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.hamcrest</groupId>
-      <artifactId>hamcrest-core</artifactId>
-      <version>${hamcrest-core.version}</version>
-      <scope>test</scope>
-    </dependency>
+<!-- DO NOT include "org.hamcrest:hamcrest-core:1.3" for test class BlockStatusProxyTest, hamcrest-core is a transitive dependency -->
```
Comment (NJtwentyone) [inline core/pom.xml:192]: I was getting error

`java.lang.NoSuchFieldError: NONE at org.hamcrest.DiagnosingMatcher.matches(DiagnosingMatcher.java:12)`

solution source:
https://stackoverflow.com/questions/27411798/eclipse-shazamcrest-java-lang-nosuchfielderror-none

File: core/src/test/java/org/pentaho/platform/web/http/api/resources/SchedulerResourceTest.java [hunk 2]
```diff
@@ -144,12 +144,8 @@ public void testCreateJobError() throws Exception {
 
     testResponse = schedulerResource.createJob( mockRequest );
     assertEquals( mockForbiddenResponse, testResponse );
-
-    verify( mockSchedulerException, times( 1 ) ).getCause();
-    verify( mockSchedulerExceptionCause, times( 1 ) ).getMessage();
```
Comment (NJtwentyone) [inline core/src/test/java/org/pentaho/platform/web/http/api/resources/SchedulerResourceTest.java:149]: checking for number times an exception.getCausse().getMessage() is to fine grain/brittle already very buildServerErrorResponse(string) gets called and verifies the correct exception argument string gets called.

File: pom.xml [hunk 3]
```diff
@@ -22,7 +22,6 @@
     <powermock.version>1.6.3</powermock.version>
     <mockito-core.version>4.0.0</mockito-core.version>
     <javax.servlet-api.version>4.0.1</javax.servlet-api.version>
-    <hamcrest-core.version>1.3</hamcrest-core.version>
```
Comment (NJtwentyone) [inline pom.xml:25]: see https://github.com/pentaho/pentaho-scheduler-plugin/pull/78/files#r1456492937

**Comment by buildguy** [comment]
### :warning: Build finished in 10m 35s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 2 test(s) failed:

<details><summary><b>org.pentaho.platform.web.http.api.resources.SchedulerResourceUtilTest.testHandlePdiScheduling_requestParamsAreTransferred</b> (click to expand)</summary><p>

```
class org.mockito.Matchers tried to access private method 'void org.mockito.internal.progress.ThreadSafeMockingProgress.<init>()' (org.mockito.Matchers and org.mockito.internal.progress.ThreadSafeMockingProgress are in unnamed module of loader 'app')
```

</p></details>

<details><summary><b>org.pentaho.platform.web.http.api.resources.SchedulerResourceUtilTest.testHandlePdiScheduling_notAllRequestParamsAreTransferred</b> (click to expand)</summary><p>

```
Could not initialize class org.mockito.Matchers
```

</p></details>

  
Tests run: **303**, Failures: **2**, Skipped: **13** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pentaho-scheduler-plugin/job/PR-78/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**